checkov/terraform/checks/resource/BaseResourceBooleanValueCheck.py

```diff
@@ -0,0 +1,25 @@
+from abc import abstractmethod
+
+import dpath
+
+from checkov.terraform.models.enums import CheckResult
+
+from checkov.terraform.checks.resource.base_check import BaseResourceCheck
+
+expected_value = True
```

Comment:
What if in some point we will need to check for `False` or maybe just generalize the class into `BaseResourceValueCheck`? which will check a certain attribute match a certain value, not just `True`
I think this should be refactored as a function of kind:
```
class BaseResourceValueCheck(...):
@abstractmethod
    def get_inspected_key(self):
        raise NotImplemented()
@abstractmethod
    def get_inspected_value(self):
        raise NotImplemented()     
```
and then implement respectively in a specific check.

Reply:
Like 👍  added